legacy/core/src/test/java/com/fsck/k9/TestApp.kt

```diff
@@ -44,4 +48,13 @@ val testModule = module {
     single { mock<NotificationStrategy>() }
     single(named("controllerExtensions")) { emptyList<ControllerExtension>() }
     single { mock<WorkManager>() }
+    single<FeatureFlagProvider> {
+        InMemoryFeatureFlagProvider(
+            featureFlagFactory = {
+                listOf(
+                    FeatureFlag(key = FeatureFlagKey("feature1"), enabled = true),
+                )
```

Comment:
I think an emptyList should be enough.

```suggestion
                emptyList<FeatureFlag>()
```